webpack.config.js

```diff
@@ -70,4 +91,85 @@ const webVitals = () => {
 	};
 };
 
-module.exports = [ webVitals ];
+/**
+ * Webpack configuration for building the plugin.
+ * Note: Only works with the `--env build=plugin` flag.
+ *
+ * @param {*} env Webpack environment
+ * @return {Object} Webpack configuration
+ */
+const buildPlugin = ( env ) => {
+	if ( 'plugin' !== env.build ) {
+		return {
+			entry: {},
+			output: {},
+		};
+	}
+
+	const from = path.resolve( __dirname );
+	const to = path.resolve( __dirname, 'build' );
+	const pluginDistIncludes = [
```

Comment:
Isn't this list essentially the inverse of the current `.gitattributes`? Since this is an allowlist it will have to be manually updated. But then there is also a `pluginDistIgnores` which is essentially the same as the `.gitattributes`? Wouldn't this be simpler to just rather go with `.distignore` instead of the `.gitattributes`?

Reply:
Humm, I agree with the negation thing since it's replicating the behavior of `.gitattributes`. TBH I wasn't much aware of `.distignore` but after exploring it I understand it's just "another" ignore file but creating distribution builds.

Reply:
Since `CopyWebpackPlugin` uses globby which provides support for such ignore files, I have switched it to use `.distignore` which is pretty much neat.